### DISCUSSION TITLE
Comment fonction acteur bouton ?

### DISCUSSION THREAD

**Discussion by cinematographes** [witsa/synapps#684] [General]
Bonjour je comprend pas comment relie un acteur bouton à une ressource wit je trouve pas le module ou mettre l'adresse de la ressource?
Merci d'avance de vos réponse

**Comment by Raphwit** [comment]
Bonjour, une documentation est disponible pour Synapps Studio à l'adresse : https://witsa.github.io/synapps/

**Reply by cinematographes** [reply]
Malheureusement il n'explique pas comment peux le relie a une ressource

**Reply by Raphwit** [reply]
Nous avons une partie liaison qui est présente, vous l'avez peut être loupée, elle se trouve à cette adresse : https://witsa.github.io/synapps/concepts/binding.html

Cette partie explique en détail le fonctionnement d'une liaison ainsi que le procédé à suivre pour effectuer des liaisons avec les fournisseurs de variable/ressource

**Reply by cinematographes** [reply]
Merci beaucoup
